website/docs/plugins/ax_sweeper.md

```diff
@@ -31,7 +31,7 @@ We include an example of how to use this plugin. The file [`example/banana.py`](
 To compute the best parameters for the Banana function, clone the code and run the following command in the `plugins/hydra_ax_sweeper` directory:
 
 ```
-python example/banana.py -m banana.x=-5:5 banana.y=-5:10.1
+python example/banana.py -m 'banana.x=int(interval(-5, 5))' 'banana.y=interval(-5, 10.1)'
```

Comment:
What are you trying to demonstrate by casting here?

Reply:
Want to highlight `int(interval(...))` construct.

Reply:
There are many things to highlight, why this?
I think it does not make sense for the example. maybe it's better to just point users to the extended grammar page to learn about all the things they can do.

Reply:
I want to highlight because this example explicitly shows the changes the default behavior. Having this example upfront will be useful for people who have been using Ax plugin.

Reply:
It this okay to keep @Omry ?

Reply:
Your call.